Lib/test/test_embed.py

```diff
@@ -343,6 +343,14 @@ def test_finalize_structseq(self):
         out, err = self.run_embedded_interpreter("test_repeated_init_exec", code)
         self.assertEqual(out, 'Tests passed\n' * INIT_LOOPS)
 
+    def test_quickened_static_code_gets_unquickened_at_Py_FINALIZE(self):
```

Comment:
Could you please specify `@support.skip_if_pgo_task` to make the PGO profiling shorter?

<br>
PS: This issue seems not reproducible for me on Win10 as far as I tried:

1. `test_finalize_structseq -R1:50`
2. `test_quickened_static_code_gets_unquickened_at_Py_FINALIZE`
3. self-contained reproducer

on the debug builds of [b733708](https://github.com/python/cpython/tree/b733708ca), [ea2f5bc](https://github.com/python/cpython/tree/ea2f5bc), [ad5e852](https://github.com/python/cpython/tree/ad5e852) and [special_for_iter2](https://github.com/sweeneyde/cpython/tree/f360d65f344e7b1e1b7c4c85369ff70a36800e7a) branches.

Reply:
Added skip_if_pgo_task.

Re: reproducibility, it's intermittent for me regardless and I don't know precisely what variables lead it to fail sometimes on the second repetition and sometimes on the 38th (etc.). IIRC I measured failure on about 4% of runs on my Win10 machine.